services/galley/test/integration/API.hs

```diff
@@ -1101,8 +1102,11 @@ listConvsPagingOk = do
   walk bill [3, 3, 3, 3, 1] -- 11 (group) + 1 (1:1) + 1 (self)
   walk carl [3, 3, 3, 3, 1] -- 11 (group) + 1 (1:1) + 1 (self)
   where
+    walk :: Foldable t => UserId -> t Int -> TestM ()
     walk u = foldM_ (next u 3) Nothing
+    next :: UserId -> Int32 -> Maybe ConvId -> Int -> TestM (Maybe ConvId)
     next u step start n = do
+      -- TODO(md): Check if the FUTUREWORK note is addressed.
       -- FUTUREWORK: support an endpoint to get qualified conversation IDs
       -- (without all the conversation metadata)
       r1 <- getConvIds u (Right <$> start) (Just step) <!! const 200 === statusCode
```

Comment:
@jschaul , I am not sure what you had in mind here. Can you please clarify?

Reply:
this futurework comment is unrelated to this PR. It is about creating a new endpoint similar to `GET ​/conversations​/ids Get all conversation IDs.` ([swagger](https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/get_conversations_ids)).